dashboard/app/views/pd/application/teacher_application_mailer/principal_approval_completed.html.haml

```diff
@@ -4,7 +4,8 @@
 
 %p
   Thank you for completing the principal approval form.
-  Your teacher’s application is now considered complete and we will complete our review soon.
+  = @application.first_name + "'s"
+  application is now considered complete and we will complete our review soon.
```

Comment:
Double complete :)

Reply:
As spec'd :upside_down_face:.  I'll update to "application is now considered complete and we will review it soon."